Cargo.toml

```diff
@@ -5,4 +5,4 @@ members = [
     "crates/explorer",
     "crates/test",
 ]
-exclude = ["examples"]
+exclude = ["examples/"]
```

Comment:
NIT: This has no effect and is just random change, right (or does clippy suggest?)? `cargo package --list` has same output. No need to change, I am just just new here, so I ask :cat:

Reply:
It has no effect, and I'm not sure how did it get there, don't think `clippy` suggested